roles/captiveportal/tasks/main.yml

```diff
@@ -53,34 +58,34 @@
     enabled: True
   when: captiveportal_enabled | bool
 
-- name: Stop uWSGI server if captive portal has been disabled
+- name: Stop uWSGI server if Captive Portal has been disabled
   systemd: 
     name: uwsgi-captiveportal.service
     state: stopped
     enabled: False
-  when: not captiveportal_enabled | bool
+  when: not captiveportal_enabled
 
-- name: Run divert to generate diversion lists for nginx
+- name: Run iiab-divert-to-nginx to generate diversion lists for NGINX
   shell: /usr/sbin/iiab-divert-to-nginx
-     
-- name: Run script to generate nginx servers from checkurls input list
+
+- name: Run iiab-make-cp-servers.py to generate NGINX servers from checkurls input list (creates /etc/nginx/sites-available/capture.conf)
   command: /usr/sbin/iiab-make-cp-servers.py
   args:
-      creates: /etc/nginx/sites-available/capture.conf
+    creates: /etc/nginx/sites-available/capture.conf
```

Comment:
rest of the roles are using nginx_conf_dir in place of sites-available|enabled

Reply:
Actually {{ nginx_config_dir }} = /etc/nginx/conf.d as currently used by most/other roles — is a different path than the above.

But this does raise the question: are all these other role are using the recommended/best path?

Reply:
I did ask that question in #2042

Reply:
@m-anish can your NGINX expert friend contribute an opinion as to whether IIAB's sites (http://box/nodered http://box/stream-fishing etc!) should place their {nodered-nginx.conf, stream-fishing.conf, etc} files into `/etc/nginx/conf.d` or `/etc/nginx/sites-available` or whatever?